### DISCUSSION TITLE
Can Export-DbaLogin export all accounts to seprate txt files?

### DISCUSSION THREAD

**Discussion by JoyWorksToo** [dataplat/dbatools#8422] [Q&A] [answered]
I need to look at  most of the logins and users on a dev server and would like to export each to its own txt file.

I need to map out users and groups they are in and keep a record of current access.

It would be helpful if I can export each account into a txt file named after the account name.

Can I do this currently or can it be added as a optional parameters?

**Comment by potatoqualitee** [comment, marked as answer]
Yes, but it would require a smidge of coding, something like

```
foreach ($login in (Get-DbaLogin -SqlInstance sql01)) {
    Export-DbaLogin -SqlInstance $login.Parent -Login $login.Name ....
}
```

This command does not properly support piping right now, sorry :/

**Reply by JoyWorksToo** [reply]
So I can make this edit myself?

Please take this as a suggestion THANK YOU
